README.md

```diff
@@ -13,15 +13,15 @@ It is a part of [Membrane Multimedia Framework](https://membraneframework.org).
 Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
 
 ```elixir
-	{:membrane_audio_mix_plugin, "~> 0.11.1"}
+  {:membrane_audio_mix_plugin, "~> 0.12.0"}
 ```
 
 ## Description
 
 Both elements operate only on raw audio (PCM), so some parser may be needed to precede them in a pipeline.
 
-Audio format can be set as an element option or received through caps from input pads. All
-caps received from input pads have to be identical and match ones in element option (if that
+Audio format can be set as an element option or received through stream format from input pads. All
+stream format received from input pads have to be identical and match ones in element option (if that
 option is different from `nil`).
```

Comment:
A simple regex isn't enough as it doesn't sound right now. BTW I realized this is a bit outdated since 0.10 as each element must send ~caps~ stream format struct, so I'd write something like

```suggestion
The elements expect to receive the same audio stream format on each input pad. It can be additionally enforced by setting an element option (`:stream_format`)
```